pmd-java/src/main/java/net/sourceforge/pmd/lang/java/symbols/internal/asm/ClassStub.java

```diff
@@ -84,12 +84,7 @@ private static boolean isValidInternalName(String internalName) {
         this.resolver = resolver;
         this.names = new Names(internalName);
 
-        this.parseLock = new ParseLock() {
-            // note to devs: to debug the parsing logic you might have
```

Comment:
The toString() method in ClassStub already as a comment about avoiding full parsing logic in debug mode. So, this comment is not necessary anymore here.